mobile/status.go

```diff
@@ -8,6 +8,9 @@ import (
 	"os"
 	"unsafe"
 
+	"github.com/status-im/status-go/extkeys"
+	"github.com/status-im/status-go/protocol/identity/ring"
```

Comment:
The import convention (and zealously enforced by the linter 😆) in this repo is to group imports alphabetically as follows:

- standard lib
- 3rd party libs
- core libs
   - ethereum/go-ethereum
   - status-im/status-go

Reply:
emm.. i thought `make lint-fix` would fix this, i was wrong, thanks for pointing out!